src/mpyl/schema/project.schema.yml

```diff
@@ -431,7 +431,27 @@ definitions:
           your ingress for old clients
         type: boolean
       priority:
-        type: number
```

Comment:
there is a pre-defined definition for this called `dtapNumberValue`

Reply:
Thanks, I wasn't aware of that! I'll apply it in the next commit 👍